resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/BasicTestsSuite.scala

```diff
@@ -81,6 +81,12 @@ private[spark] trait BasicTestsSuite { k8sSuite: KubernetesSuite =>
       })
   }
 
+  test("Run SparkPi with spark.io.compression.codec=snappy for SPARK-26995.", k8sTestTag) {
```

Comment:
Sorry to be a pain about these things, but could you instead change an existing test?

Integration tests take much longer to run than unit tests, the more we can test with the same invocation, the better.

Reply:
Sure, no problem.